easybuild/easyconfigs/o/OpenMPI/OpenMPI-1.8.4-iccifort-2015.1.133-GCC-4.9.2.eb

```diff
@@ -16,6 +16,7 @@ configopts = '--with-threads=posix --enable-shared --enable-mpi-thread-multiple
 configopts += '--enable-mpirun-prefix-by-default '  # suppress failure modes in relation to mpirun path
 configopts += '--with-hwloc=$EBROOTHWLOC '  # hwloc support
 configopts += '--disable-dlopen '  # statically link component, don't do dynamic loading
+configopts += ' CC=icc F77=ifort FC=ifort CXX=icpc '
```

Comment:
EasyBuild sets these in the environment, when building/installing something using the `iccifort` toolchain.

I'm not sure how this can fix anything...

@pescobar: are you saying that `mpifort` picks up `ifort` if `FC=ifort` was passed on the configure command line (which is different from just having `$FC` set to `ifort`)?

Are you sure you didn't have `$FC` defined when obtaining `ifort` via `mpifort --version`?